lib/WP_Auth0_DBManager.php

```diff
@@ -294,6 +294,54 @@ public function install_db( $version_to_install = null, $app_token = '' ) {
 			}
 		}
 
+		// 3.6.0
+
+		if ( ( $this->current_db_version < 18 && 0 !== $this->current_db_version ) || 18 === $version_to_install ) {
+			if ( $options->get( 'passwordless_enabled', FALSE ) ) {
+				$pwl_method = $options->get( 'passwordless_method' );
+				switch ( $pwl_method ) {
+
+					// SMS passwordless just needs 'sms' as a connection
+					case 'sms' :
+						$options->set( 'lock_connections', 'sms' );
+						break;
+
+					// Social + SMS means there are existing social connections we want to keep
+					case 'socialOrSms' :
+						$options->add_lock_connection( 'sms' );
+						break;
+
+					// Email link passwordless just needs 'email' as a connection
+					case 'emailcode' :
+					case 'magiclink' :
+						$options->set( 'lock_connections', 'email' );
+						break;
+
+					// Social + Email means there are social connections be want to keep
+					case 'socialOrMagiclink' :
+					case 'socialOrEmailcode' :
+						$options->add_lock_connection( 'email' );
+						break;
+				}
+
+				if ( in_array( $pwl_method, array( 'emailcode', 'socialOrEmailcode' ) ) ) {
+					$lock_json = trim( $options->get( 'extra_conf' ) );
+					if ( empty( $lock_json ) ) {
```

Comment:
The second parameter in that `get` method isn't used if it's empty, only if it's not set. If we're doing an upgrade then it's set to something so that value will not be used. 

Still, could be improved ... fixed!